pkg/monitortests/testframework/uploadtolokiserializer/monitortest.go

```diff
@@ -192,12 +192,12 @@ func UploadIntervalsToLoki(intervals monitorapi.Intervals) error {
 	for _, i := range intervals {
 		logLine := map[string]string{
 			"_entry": i.Message,
+			"from":   i.From.String(),
+			"to":     i.To.String(),
```

Comment:
Hmm this looks like it will label the entries in loki and positively explode the cardinality, probably taking all of loki down. You could append to the something to the i.Message for now. I hope to get these logged as json sometime soon anyhow.

Reply:
Nice catch!  I will append to the i.Message.